deltacat/compute/stats/models/stats_result.py

```diff
@@ -86,7 +85,8 @@ def merge(stats_list: List[StatsResult],
         merged_stats: Dict[StatsType, int] = defaultdict(int)
         for stats_result in stats_list:
             for stat_type in stats_to_collect:
-                merged_stats[stat_type.value] += stats_result[stat_type.value]
+                if stats_result:
```

Comment:
Yeah, I remembered I saw the line here break because of stats_result being None before. I think it's because I'm rebatching deltas now. The list of StatsResults for each manifest file needs some merging or splitting when resolving to it's original delta.